cmd/bugzilla-config-manager/main.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/test-infra/prow/plugins"
+	utilpointer "k8s.io/utils/pointer"
+)
+
+func main() {
+	// TODO: Fetch oracle, run(), createPR
```

Comment:
Managers themselves will not maintain their PRs - just commits. This is to allow non-Go managers to participate.

DPTP-2443 is the story for the helper to manager PRs. I'm not super-strong on whether the manager should make commits or not, but I'm leaning towards yes, because it knows what changes it makes.

Reply:
It might be easier to test if it doesn't though and the helper tool could just use "Ran $configmanager name\n\n$OUTPUT" for the commit message I think?

Reply:
Certainly, it's a tradeoff for the opportunity to have better commit messages, because the managers know what they are actually enforcing - but I'm not sure if that's valuable enough. I don't have strong opinion about it. Let's start with the easier route (yours).